kolibri/plugins/coach/assets/src/views/lessons/LessonSummaryPage/index.vue

```diff
@@ -0,0 +1,483 @@
+            :key="resourceId"
+            v-for="(resourceId, index) in workingResources"
+          >
+            <td class="core-table-icon-col">
+              <ui-icon-button
+                type="flat"
+                icon="keyboard_arrow_up"
+                :ariaLabel="$tr('moveResourceUpButtonDescription')"
+                :disabled="index === 0"
+                @click="moveUpOne(index)"
+                class="position-adjustment-button"
+              />
+              <ui-icon-button
+                type="flat"
+                icon="keyboard_arrow_down"
+                :ariaLabel="$tr('moveResourceDownButtonDescription')"
+                :disabled="index === (workingResources.length - 1)"
+                @click="moveDownOne(index)"
+                class="position-adjustment-button"
+              />
+            </td>
+            <td>
+              {{ resourceTitle(resourceId) }}
+            </td>
+            <td>
+              <!-- TODO stubbed. Need progress endpoint that scopes by user -->
+              <progress-bar
+                v-if="false"
+                class="resource-progress-bar"
+                :progress="resourceProgress(resourceId)"
+                :showPercentage="false"
+              />
+            </td>
+            <td>
+              <k-button
+                :text="$tr('resourceRemovalButtonLabel')"
+                @click="stageRemoval(index, resourceId)"
+                appearance="flat-button"
+              />
+            </td>
+          </tr>
+        </transition-group>
+      </core-table>
+
+      <manage-lesson-modals
+        :currentAction="currentAction"
+        @cancel="currentAction=null"
+      />
+    </div>
+
+  </div>
+
+</template>
+
+
+<script>
+
+  import dropdownMenu from 'kolibri.coreVue.components.dropdownMenu';
+  import kButton from 'kolibri.coreVue.components.kButton';
+  import uiIconButton from 'keen-ui/src/UiIconButton';
+  import progressBar from 'kolibri.coreVue.components.progressBar';
+  import kRouterLink from 'kolibri.coreVue.components.kRouterLink';
+  // why PascalCase?
+  import coreTable from 'kolibri.coreVue.components.CoreTable';
+  import map from 'lodash/map';
+  import ManageLessonModals from '../ManageLessonModals';
+  import { LessonActions, CollectionTypes } from '../../../lessonsConstants';
+  import StatusIcon from '../StatusIcon';
+  import contentIcon from 'kolibri.coreVue.components.contentIcon';
+  import InfoIcon from '../InfoIcon';
+  import { selectionRootLink } from '../lessonsRouterUtils';
+  import { createSnackbar, clearSnackbar } from 'kolibri.coreVue.vuex.actions';
+  import { saveLessonResources, updateCurrentLesson } from '../../../state/actions/lessons';
+  import debounce from 'lodash/debounce';
+
+  const removalSnackbarTime = 5000;
+  const saveDebounceTime = 6000;
+
+  // debounced function, pulling out of direct binding to preserve cancel() function
+  const commitRemovals = debounce(function() {
```

Comment:
I use the `cancel()` method that lodash bakes in to reset the timer every time something new is deleted - call corresponds to the amount of time the snackbar is visible